odl/tomo/geometry/parallel.py

```diff
@@ -1014,9 +1196,9 @@ def frommatrix(cls, apart, dpart, init_matrix):
         # Use the standard constructor with these vectors
         axis, det_pos, det_axis_0, det_axis_1 = transformed_vecs
         if translation.size == 0:
-            kwargs = {}
+            pass
```

Comment:
Sure, this was the work-in-progress version just in case I want to switch back. Forgot to change to final.